virtual_ecosystem/models/hydrology/hydrology_model.py

```diff
@@ -398,6 +446,7 @@ def _update(self, time_index: int, **kwargs: Any) -> None:
         * evapotranspiration (current time step), [mm]
         * accumulated surface runoff (previous time step), [mm]
         * accumulated subsurface flow (previous time step), [mm]
+        * aerodynamic_resistance_canopy, [s m-1]
```

Comment:
add description of canopy evaporation, drip to docstring here. Also, add TODO to update description of matric potential and vertical flow